supporting_content/columns/servicesubcategory.md

```diff
@@ -0,0 +1,210 @@
+| Web                       | Application Frameworks            |                                                                           | App Configuration, Maps, SignalR Service, Fluid Relay                                                                              | Maps {Directions API, Distance Matrix API, Geolocation API, etc.}  |
+| Web                       | Web Platforms                     | Elastic Beanstalk                                                         | App Service, Spring Apps                                                                                                           | App Engine                                                         |
+| Web                       | Other (Web)                       | Lightsail                                                                 |                                                                                                                                    |                                                                    |
+| Other                     | Other (Other)                     |                                                                           |                                                                                                                                    |
+
+### Principles
+
+In addition to the principles as defined during the creation of Service Categories in FOCUS 0.5, the following principles shall also be followed when constructing Service Subcategories:
+
+- Undercategorize rather than overcategorize where there is not broad consensus and/or materiality.
+  - Allow the community to advocate for new Subcategories (and Categories) in future releases, and leave Services uncategorized and/or unsubcategorized in the meantime.
+- Assign each Service to the one (and only one) Subcategory that describes its primary function.  
+  - The Service hierarchy is meant to serve as a relatively high-level set of groupings that primarily address the needs of the Executive persona.  A given Service can fulfill multiple functions across and within organizations, and so the assignment of a Service to one and only one Category and Subcategory may be inaccurate from org to the next.  However, this hierarchy will then give the Practitioner a solid place to start, and they can make a few tweaks to its composition to meet their org's needs rather than create a hierarchy from scratch.  Ultimately, the creation and maintenance of the Service hierarchy requires us to embrace the imperfect and somewhat subjective nature of assigining a given Service to a single Category / Subcategory.
+  - The (forthcoming) SKU hierarchy will be an opportunity to get very specific on the categorizations of cost and usage activity.  For example, GCP maintains a seven-level taxonomy for every SKU, and this will hopefully more concretely address the lower-level reporting needs of the Engineering persona.
+
```

Comment:
```suggestion
- Use the "Platforms" suffix whenever a subcategory is needed to cover products and services that span multiple other subcategories.
  - Do not define a "Platforms" subcategory if the service category does not have at least 2 other subcategories.

```

Reply:
Accepted in a separate commit.  Resolving.